### PR TITLE
 [JENKINS-53285] Update plugins to latest to avoid warnings

### PR DESCRIPTION
**Pull request by batmat**
https://issues.jenkins-ci.org/browse/JENKINS-53285 this should already reduce the number of warnings we receive in Sentry.

cc @jglick

### CODE REVIEW

File: services/essentials.yaml [hunk 1]
```diff
@@ -69,6 +69,33 @@ spec:
     - groupId: io.jenkins.plugins
       artifactId: artifact-manager-s3
       version: '1.1'
+    - groupId: org.jenkins-ci.plugins
+      artifactId: authentication-tokens
+      version: '1.3'
+    - groupId: org.jenkins-ci.plugins
+      artifactId: conditional-buildstep
```
Comment (rtyler) [inline services/essentials.yaml]: This is no longer required and shouldn't be here.

File: services/essentials.yaml [hunk 2]
```diff
@@ -69,6 +69,33 @@ spec:
...
+      version: 1.3.6
+    - groupId: org.jenkins-ci.plugins.icon-shim
+      artifactId: icon-shim
+      version: 2.0.3
+    - groupId: org.jenkins-ci.plugins
+      artifactId: javadoc
```
Comment (rtyler) [inline services/essentials.yaml]: This is also no longer required.

File: services/essentials.yaml [hunk 3]
```diff
@@ -69,6 +69,33 @@ spec:
...
+      version: '1.4'
+    - groupId: org.jenkins-ci.plugins
+      artifactId: jsch
+      version: 0.1.54.2
+    - groupId: org.jenkins-ci.plugins
+      artifactId: mapdb-api
```
Comment (rtyler) [inline services/essentials.yaml]: This is also no longer required

File: services/essentials.yaml [hunk 4]
```diff
@@ -69,6 +69,33 @@ spec:
...
+      version: 1.0.9.0
+    - groupId: org.jenkins-ci.main
+      artifactId: maven-plugin
+      version: 3.1.2
+    - groupId: org.jenkins-ci.plugins
+      artifactId: node-iterator-api
+      version: 1.5.0
+    - groupId: org.jenkins-ci.plugins
+      artifactId: run-condition
```
Comment (rtyler) [inline services/essentials.yaml]: This is also no longer required

File: services/essentials.yaml (hunk 2 above)
Comment (rtyler) [reply services/essentials.yaml]: Looks like this is required by the newer maven plugin

File: services/essentials.yaml [hunk 5]
```diff
@@ -69,6 +69,21 @@ spec:
...
+      artifactId: icon-shim
+      version: 2.0.3
+    - groupId: org.jenkins-ci.plugins
+      artifactId: javadoc
+      version: '1.4'
+    - groupId: org.jenkins-ci.plugins
+      artifactId: jsch
+      version: 0.1.54.2
+    - groupId: org.jenkins-ci.main
+      artifactId: maven-plugin
```
[the remark this answers is not in the document]
Comment (batmat) [reply services/essentials.yaml]: I think I tried in the past, but not recently. The whole world was still depending on it, and I had filed https://issues.jenkins-ci.org/browse/JENKINS-46684 back then. I will need to recheck, possibly now the situation improved. Looking quickly.

File: services/essentials.yaml [hunk 6]
```diff
@@ -69,6 +69,21 @@ spec:
     - groupId: io.jenkins.plugins
       artifactId: artifact-manager-s3
       version: '1.1'
+    - groupId: org.jenkins-ci.plugins
+      artifactId: authentication-tokens
+      version: '1.3'
+    - groupId: org.jenkins-ci.plugins.icon-shim
+      artifactId: icon-shim
```
[the remark this answers is not in the document]
Comment (batmat) [reply services/essentials.yaml]: So, I just tried, and it's pulled in by something else. (didn't check what yet). And if so, it's the 1.0.3 that get taken, so this must stay at least for now.

File: services/essentials.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (batmat) [reply services/essentials.yaml]: Yay! Enriched and repurposed a bit https://github.com/jenkins-infra/evergreen/pull/208 to include this. Thanks for checking Jesse!

**Pull request merged**